nibabel/freesurfer/io.py

```diff
@@ -160,6 +160,29 @@ def read_morph_data(filepath):
     return curv
 
 
+def write_morph_data(filepath, values):
+    """Write out a Freesurfer morphometry data file.
+
+    See:
+    http://www.grahamwideman.com/gw/brain/fs/surfacefileformats.htm#CurvNew
+
+    Parameters
+    ----------
+    filepath : str
+        Path to annotation file to be written
+    values : ndarray, shape (n_vertices,)
+        Surface morphometry values
+    """
+    magic_bytes = np.array([255, 255, 255], dtype=np.uint8)
+    with open(filepath, 'wb') as fobj:
```

Comment:
`np.array([255, 255, 255], dtype=np.uint8).tofile(fobj)` for such a `fobj` results in:

Python 2.7:

    IOError: first argument must be a string or open file

Python 3:

    OSError: first argument must be a string or open file

(Numpy 1.8.2, in both cases.)